packages/examples/todo/src/redux/store.ts

```diff
@@ -4,13 +4,13 @@ import { logger } from './logger'
 import { proxyReducer } from './reducers'
 
 const initialState = {
-  visibilityFilter: VisibilityFilter.ALL,
-  todoList: [],
-  todoMap: {},
+  root: {
+    visibilityFilter: VisibilityFilter.ALL,
+    todoList: [],
+    todoMap: {},
+  },
 }
```

Comment:
This PR showcases the awkwardness of working with the Repo API when all we need is a single document. See #40